### DISCUSSION TITLE
Ben I have met a break threw on the dev mode on school laptops !!!!!!!!

### DISCUSSION THREAD

**Discussion by purpleguy5728** [TechItAllServices/TechItAllServices-Main#7] [General]
![Screenshot 2024-05-26 12 12 55 PM](https://github.com/TechItAllServices/TechItAllServices-Main/assets/145482556/496571c3-d0f3-4077-b33f-71ad54e4318b)
![Screenshot 2024-05-26 12 50 44 PM](https://github.com/TechItAllServices/TechItAllServices-Main/assets/145482556/cd1579cb-7ea2-4678-9f3e-9b27c5a50b57)

**Comment by purpleguy5728** [comment]
@TechItAll

**Comment by TechItAll** [comment]
@purpleguy5728 What did you do? Are you sure thats a school chromebook?

**Comment by purpleguy5728** [comment]
yes I'm on the my school account on a chrome book at home

**Comment by TechItAllSA** [comment]
The attempt was on a Non Enrolled PC but Enrolled Google Account. Attempt invalid.
